src/interfaces/IAToken.sol

```diff
@@ -0,0 +1,96 @@
+/** 
+    Created by Arcadia Finance
+    https://www.arcadia.finance
+
+    SPDX-License-Identifier: AGPL-3.0-or-later
+ */
+ 
+pragma solidity >=0.4.22 <0.9.0;
+
+import {IERC20} from './IERC20.sol';
+import {IScaledBalanceToken} from './IScaledBalanceToken.sol';
+
+interface IAToken is IERC20, IScaledBalanceToken {
```

Comment:
Why do we need an interface with all functions?
Would limit it to the ones we actually use?

Reply:
We don't even need the interface now tbh. Since we only might aswel  remove these files